Src/DeUrgenta.RecurringJobs/appsettings.json

```diff
@@ -30,6 +30,10 @@
     "NotificationCleanupJobConfig": {
       "IsEnabled": "true",
       "CronExpression": "0 6 * * *"
+    },
+    "EventArchivalJobConfig": {
+      "IsEnabled": "true",
+      "CronExpression": "0 0 1 * * ?"
```

Comment:
Please use https://crontab.guru/ to generate the CRON expression so that all the crons are consistent